packages/node/test/integrations/localvariables.test.ts

```diff
@@ -244,7 +244,25 @@ describe('LocalVariables', () => {
     expect(eventProcessor).toBeUndefined();
   });
 
-  it.only('Should cache identical uncaught exception events', async () => {
```

Comment:
Ooops, this only shouldn't have been there!

Reply:
good catch!

Reply:
I think there's a lint for this 🤔